engine/api/README.md

```diff
@@ -3,23 +3,23 @@
 This is the core component of CDS.
 To start CDS api, the only mandatory dependency is a PostgreSQL database and a path to the directory containing other CDS binaries.
 
-There is are two ways to set up CDS:
+There are two ways to set up CDS:
 
-- as [toml](https://github.com/toml-lang/toml) configuration
-- over environment variables.
+- with [toml](https://github.com/toml-lang/toml) configuration
+- with environment variables.
 
 ## CDS API Third-parties
 
-At the minimum, CDS needs a PostgreSQL Database >= 9.4. But for serious usage your may need :
+At the minimum, CDS needs a PostgreSQL Database >= 9.4. But for serious usage your may need:
 
-- A [Redis](https://redis.io) server or sentinels based cluster used as a cache and session store
-- A LDAP Server for authentication
-- A SMTP Server for mails
+- A [Redis](https://redis.io) server or sentinels based cluster used as cache and session store
+- An LDAP Server for authentication
```

Comment:
http://www.grammar.com/a-vs-an-when-to-use/

https://en.wikipedia.org/wiki/Lightweight_Directory_Access_Protocol  <- ctrl+f  "an LDAP" vs. "a LDAP"  ;)